types/google-one-tap/index.d.ts

```diff
@@ -0,0 +1,58 @@
+// Type definitions for google-one-tap 1.0
+// Project: https://developers.google.com/identity/one-tap/web
+// Definitions by: voidpumpkin <https://github.com/voidpumpkin>
+//                 kostasmanionis <https://github.com/kostasmanionis>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace google;
```

Comment:
If these types are consumable globally, there should be global tests as well.

https://github.com/definitelyTyped/DefinitelyTyped/#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module